.buildkite/auditbeat/auditbeat-pipeline.yml

```diff
@@ -86,6 +86,12 @@ steps:
         artifact_paths:
           - "auditbeat/build/*.xml"
           - "auditbeat/build/*.json"
+        plugins:
+          - test-collector#v1.10.2:
+              files: "auditbeat/build/TEST-*.xml"
+              format: "junit"
+              # branches: "main"
```

Comment:
I assume the branch is commented out for testing in the PR, but don't we want it to only be on main (and maybe 8.*) branches in official runs?